python/pylibcudf/pylibcudf/tests/common/utils.py

```diff
@@ -242,13 +243,20 @@ def is_nested_list(typ):
     return nesting_level(typ)[0] > 1
 
 
-def _convert_numeric_types_to_floating(pa_table):
+def _convert_types(pa_table, input_pred, result_type):
     """
     Useful little helper for testing the
     dtypes option in I/O readers.
 
     Returns a tuple containing the pylibcudf dtypes
     and the new pyarrow schema
+
+    Parameters
+    ----------
+    input_pred : function
```

Comment:
Missing parameter docstring.
```suggestion
    pa_table :
        PyArrow table
    input_pred : function
```